lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -51,15 +53,37 @@ module.exports = function({workspace, options}) {
 		await workspace.write(manifestResource);
 	}
 
+	function readManifestMinUI5Version() {
+		return workspace.byPath(`${pathPrefix}/manifest.json`).then((manifestResource) => {
+			if (manifestResource) {
+				return manifestResource.getBuffer().then((buffer) => {
+					return JSON.parse(buffer.toString());
+				});
+			}
+			return {};
+		}).then((manifestContent) => {
+			manifestContent["sap.ui5"] = manifestContent["sap.ui5"] || {};
+			manifestContent["sap.ui5"].dependencies = manifestContent["sap.ui5"].dependencies || {};
+			return manifestContent["sap.ui5"].dependencies.minUI5Version = manifestContent["sap.ui5"].dependencies.minUI5Version || "";
+		});
+	}
+
 	log.verbose("Collecting flexibility changes");
-	return workspace.byGlob(`${pathPrefix}/changes/*.change`)
+	return workspace.byGlob(`${pathPrefix}/changes/*.{change,variant,ctrl_variant,ctrl_variant_change,ctrl_variant_management_change}`)
 		.then((allResources) => {
-			return flexChangesBundler({
-				resources: allResources,
-				options: {
-					namespace: options.namespace,
-					pathPrefix: pathPrefix
+			return readManifestMinUI5Version().then((version) => {
+				let hasFlexBundleVersion = false;
+				if (parseFloat(version) >= 1.73) {
```

Comment:
In other modules (lib/processors/manifestCreator.js), we're using the "semver" module to parse and compare versions. For the limited use case here, using parseFloat might be sufficient, however.

Reply:
If it is ok, will keep it as it is. If there will be in further another version than we will add semver.